spec/models/benefitting_country_spec.rb

```diff
@@ -170,3 +186,36 @@
     end
   end
 end
+
+RSpec.describe BenefittingCountry::Region do
```

Comment:
I wonder now, if `BenefittingCountry::Region` is becoming useful on its own, if we should break it out into its own file (and, conversely, test suite)

Reply:
Do it 👍

Reply:
https://trello.com/c/8vPgSZyH